src/gui/optionsdlg.cpp

```diff
@@ -173,6 +173,9 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     m_ui->groupFileAssociation->setVisible(false);
 #endif
 
+    m_ui->textWebUIRootFolder->setMode(FileSystemPathEdit::Mode::DirectoryOpen);
+    m_ui->textWebUIRootFolder->setDialogCaption(tr("Choose Alternative UI source path"));
```

Comment:
It is not clear what does "source" mean here. I suggest something like "Choose Alternative UI files location".